benchmark/pkg/scraper/scraper_test.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scraper
+
+import (
+	"reflect"
+	"testing"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+func TestGetMetricsForRun(t *testing.T) {
+	apiCallLatencies, podStartupLatency, err := GetMetricsForRun("ci-kubernetes-e2e-gce-scalability", 1937)
```

Comment:
Run 1937 will disappear at some point. Please just download it, store it somewhere and run it locally.